test/merge/basics.wast.combined.finalized.opt

```diff
@@ -7,12 +7,8 @@
  (import "env" "some-collide" (func $some-collide))
  (import "env" "some-func-b" (func $some-func-b))
  (import "env" "some-collide" (func $some-collide$0))
- (global $global-collide i32 (i32.const 0))
```

Comment:
Hmm, this optimization is too good, it makes some testcases less interesting ;)

How about changing these merge testcase globals to be mutable, so they aren't optimized out?

Reply:
Ah, right, made it so that there's a mutable and a constant global for each one.